caps/templates/council_detail.html

```diff
@@ -48,23 +48,29 @@ <h2>Other nearby councils</h2>
     </div>
     <div class="col-md-7 col-lg-8">
 
-      {% comment %}
-        <div class="card ceuk-card ceuk-card--emphasised ceuk-card--yellow mb-gutter">
+        <div class="card ceuk-card ceuk-card--emphasised ceuk-card--orange mb-gutter">
             <div class="card-header">
                 <h2>Headline promises</h2>
             </div>
             <div class="card-body">
-                <blockquote class="blockquote mt-n2">
-                    <p class="mb-0">“carbon neutral for our direct emissions by 2025”</p>
-                    <footer class="blockquote-footer"><cite title="Source Title">Mayor’s Climate Emergency Action Plan, 2019</cite></footer>
-                </blockquote>
-                <blockquote class="blockquote mb-0">
-                    <p class="mb-0">“carbon neutral and climate reslilent by 2030”</p>
-                    <footer class="blockquote-footer"><cite title="Source Title">Bristol One City Climate Policy, 2020</cite></footer>
+                {% for promise in council.promise_set.all %}
```

Comment:
I think the logic for this section is wrong. Eg: when a council has both a promise and a "no promise" line in the source spreadsheet (like Allerdale right now):

![image](https://user-images.githubusercontent.com/739624/133439329-fd06c337-ff44-4fa3-98f2-a6b0e5b59230.png)

I tried fixing it but couldn’t get the right combination of booleans in the `{% if %}` tag!

As mentioned in Slack, I think it should go something like:

```
{% if promise_set_is_not_empty and first_promise_isnt_explicit_no_promise %}
  {% for each promise in promise_set %}
    {# … print the blockquote etc #}
  {% endfor %}
{% else %}
    <p>
        We haven’t yet found any carbon zero target dates for this council.
    </p>
    <p class="text-muted mb-0">
        Have you found one? <a href="{% url 'about' %}">Let us know!</a>
    </p>
{% endif %}
```

Note the HTML in the `{% else %}` tag is different to currently, and is also the same regardless of whether the we’ve checked the council for promises or not.

Eventually we’ll want to link to a Google Form or something for people to suggest improvements, but for now, a link to the About page (same as the "Alpha" banner at the bottom of the site) is better than nothing.